src/wlapplication_messages.cc

```diff
@@ -19,178 +19,207 @@
-	          << endl
-	          /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but */
-	          /** TRANSLATORS: it HAS TO BE CONSISTENT with the translation in the widelands
-	             textdomain */
-	          << _(" --dock_windows_to_edges=[true|false]\n"
-	               "                      Eliminate a window’s border towards the\n"
-	               "                      edge of the screen when the edge of the\n"
-	               "                      window is next to the edge of the screen.")
-	          << endl
-	          << _(" --panel_snap_distance=[0 ...]\n"
-	               "                      Move a window to the edge of the panel when\n"
-	               "                      the edge of the window comes within this\n"
-	               "                      distance from the edge of the panel.")
-	          << endl
-	          << _(" --snap_windows_only_when_overlapping=[yes|no]\n"
-	               "                      Only move a window to the edge of a panel\n"
-	               "                      if the window is overlapping with the\n"
-	               "                      panel.")
-	          << endl
-	          << endl;
-	std::cout << _(" --verbose            Enable verbose debug messages") << endl << endl;
-	std::cout << _(" --help               Show this help") << endl << endl;
-	std::cout
-	   << _(" <save.wgf>/<replay.wrpl> \n"
-	        "                      Directly loads the given savegame or replay. Useful for\n"
-	        "                      .wgf/.wrpl file extension association. Does not work with\n"
-	        "                      other options. Also see --loadgame/--replay.")
-	   << endl
-	   << endl;
-	std::cout << _("Bug reports? Suggestions? Check out the project website:\n"
-	               "        https://www.widelands.org/\n\n"
-	               "Hope you enjoy this game!")
-	          << endl;
+
+	print_help_option(
+	   _("--localedir=DIRNAME"), _("Use specified directory for the Widelands locale files."));
+	print_help_option(
+	   /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but it HAS TO BE
+	      CONSISTENT with the translation in the widelands textdomain */
+	   _("--coredump=[true|false]"),
+	   _("Generates a core dump on segfaults instead of using the SDL."));
+	print_help_option(_("--language=[de_DE|sv_SE|...]"), _("The locale to use."));
+	print_help_option(
+	   /** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but it HAS TO BE
+	      CONSISTENT with the translation in the widelands textdomain. */
+	   /** TRANSLATORS: A syncstream is a synchronization stream. Syncstreams are used in multiplayer
+	      games to make sure that there is no mismatch between the players. */
+	   _("--write_syncstreams=[true|false]"),
+	   _("Create syncstream dump files to help debug network games."));
+	print_help_option(_("--autosave=[...]"), _("Automatically save each n minutes."));
```

Comment:
Is this a list? Maybe better `--autosave=n`. A reader can associate then between 'n' as value and the 'n' in the description.

Same below.